src/util/generatePreview.js

```diff
@@ -38,6 +39,19 @@ const errorHandlerScript = `(${(() => {
   };
 }).toString()}());`;
 
+const alertReplacementScript = `(${(() => {
+  // Object.create so our reference
+  // doesn't get destroyed when we delete it on the window
+  const _swal = Object.create({swal: window.swal});
+
+  Reflect.defineProperty(window, 'alert', {
```

Comment:
`Reflect` is cool, but it’s [not supported by IE or current Safari](http://kangax.github.io/compat-table/es6/#test-Reflect). Since this code is running in the preview environment and thus doesn’t have the benefit of our Babel polyfill, I think we’ll need to use `Object.defineProperty`